src/themes/oneui/base-variables/_misc.scss

```diff
@@ -1,5 +1,14 @@
 $transition-duration: .3s;
 
+/**
```

Comment:
this is good, but we will never remember this is here. Can it be moved to storybook?